docs/pages/bssw/bssw_styling_curated.md

```diff
@@ -15,7 +15,7 @@ article can highlight several types of resources, including the following: book,
 
 The following figure shows different parts of a curated content article.
 
-- <img src='https://github.com/betterscientificsoftware/images/raw/master/documentation-cc-example.jpg'/>
+- <img src='https://github.com/betterscientificsoftware/bssw.io/raw/master/images/documentation-cc-example.jpg'/>
```

Comment:
This should refer to the file in the working tree, not to a URL, like the comment on bssw_styling_common.md

Reply:
> This should refer to the file in the working tree, not to a URL, like the comment on bssw_styling_common.md

@bernhold, it can't because this is a GitHub Pages Jekyll site.  It has to be the full URL.  Try changing this to a relative path and build the Jekyll site yourself locally and see what happens (the link will be broken on the generated `*.html` page).

Reply:
Sorry, now I see what this is. This is goo.  Thanks.